Nataly_Bondarenko/test_framework/conftest.py

```diff
@@ -1,12 +1,22 @@
 import pytest
+import json
 
+from Auto_Lessons.HomeWork.Nataly_Bondarenko.test_framework.CONSTANTS import ROOT_DIR
 from Auto_Lessons.HomeWork.Nataly_Bondarenko.test_framework.page_objects.item_page import ItemPage
 from Auto_Lessons.HomeWork.Nataly_Bondarenko.test_framework.page_objects.login_page import LoginPage
 from Auto_Lessons.HomeWork.Nataly_Bondarenko.test_framework.page_objects.main_page import MainPage
 from Auto_Lessons.HomeWork.Nataly_Bondarenko.test_framework.utilities.config_parser import ReadConfig
+from Auto_Lessons.HomeWork.Nataly_Bondarenko.test_framework.utilities.configurations import Configurations
 from Auto_Lessons.HomeWork.Nataly_Bondarenko.test_framework.utilities.driver_factory import DriverFactory
 
 
+@pytest.fixture()
+def env():
+    with open(f'{ROOT_DIR}/configurations/configurations.json') as file:
+        env_dict = json.loads(file.read())
+    return Configurations(**env_dict)
+
+
 @pytest.fixture()
 def create_driver():
     driver = DriverFactory.create_driver(driver_id=ReadConfig.get_browser_id())
```

Comment:
your create driver should use your new JSON approach

Reply:
fixed